notifications/16102019-tutorialOffer.yml

```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
```

Comment:
@irgolic is that OK? So the notification will appear in the first 5 startsups of Orange?

Reply:
Yes, until it is responded to (either accept or reject is pressed, as opposed to dismiss).